src/test/java/hudson/plugins/parameterizedtrigger/test/TriggerBuilderTest.java

```diff
@@ -211,6 +220,36 @@ public void testBlockingTriggerWithDisabledProjects() throws Exception {
                 "Waiting for the completion of project3");
     }
 
+    /** Verify that workflow build can be triggered */
+    public void testTriggerWithWorkflow() throws Exception {
+        WorkflowJob p = (WorkflowJob) Jenkins.getInstance().createProject(WorkflowJob.class, "project1");
+        p.setDefinition(new CpsFlowDefinition("println('hello')"));
+
+        Project<?, ?> triggerProject = createFreeStyleProject("projectA");
+        TriggerBuilder triggerBuilder = new TriggerBuilder(createTriggerConfig("project1"));
+        triggerProject.getBuildersList().add(triggerBuilder);
+        triggerProject.scheduleBuild2(0).get();
+
+        assertNotNull(triggerProject.getLastBuild());
+        assertEquals(1, p.getBuilds().size());
+        assertNotNull(p.getLastBuild());
```

Comment:
Does this actually test anything different from `BuildTriggerConfigTest.testBuildWithWorkflowProjects`?